### PR TITLE
[dev-v2.7] Add Longhorn 1.5.5 chart

### PR DESCRIPTION
**Pull request by mantissahz**
Release Longhorn v1.5.5 into Rancher 2.7

Longhorn https://github.com/longhorn/longhorn/issues/8356
Rancher https://github.com/rancher/rancher/issues/44762

cc @innobead @PhanLe1010 @khushboo-rancher @roger-ryao

Waiting for Longhorn 1.5.5 images merged into the Rancher image-mirror list.

### CODE REVIEW

**Review by PhanLe1010 — APPROVED** [review]
LGTM

Could QA help to test @khushboo-rancher ?

**Review by PhanLe1010 — APPROVED** [review]

**Review by shuo-wu — APPROVED** [review]

**Review by lucasmlp — APPROVED** [review]
LGTM 🚀

File: packages/longhorn/longhorn-1.5/generated-changes/patch/values.yaml.patch [hunk 1]
```diff
@@ -6,31 +6,31 @@
      engine:
 -      repository: longhornio/longhorn-engine
 +      repository: rancher/mirrored-longhornio-longhorn-engine
```
Comment (lucasmlp) [inline packages/longhorn/longhorn-1.5/generated-changes/patch/values.yaml.patch:8]: Here the images were patched to use the mirrored ones.

File: packages/longhorn-crd/longhorn-1.5/generated-changes/exclude/values.yaml [hunk 2]
```diff
@@ -30,25 +30,25 @@ image:
   longhorn:
     engine:
       repository: longhornio/longhorn-engine
```
Comment (lucasmlp) [inline packages/longhorn-crd/longhorn-1.5/generated-changes/exclude/values.yaml:32]: Here the images are not the mirrored ones.

Comment (PhanLe1010) [reply packages/longhorn-crd/longhorn-1.5/generated-changes/exclude/values.yaml:32]: Oh. This one is in `exclude` folder which means the whole file will be removed. I think it shouldn't affect anything. WDYT?

**Comment by khushboo-rancher** [comment]
Validation - PASSED

Validated with Rancher 2.7.12, installation, uninstallation, upgrade and p0 test cases for volume worked fine. I didn't observe any longhornio images in the deployment, all the components are deployed with mirrored images.

**Pull request merged**